R/driver_rds_remote.R

```diff
@@ -0,0 +1,199 @@
+R6_driver_rds_remote <- R6::R6Class(
+  "driver_rds_s3",
+  public = list(
+    ## TODO: things like hash_algorithm: do they belong in traits?
+    ## This needs sorting before anyone writes their own driver!
+    file_ops = NULL,
+    path = NULL,
+    compress = NULL,
+    mangle_key = NULL,
+    mangle_key_pad = NULL,
+    hash_algorithm = NULL,
+    traits = list(accept = "raw"),
+
+    initialize = function(file_ops, path, compress, mangle_key, mangle_key_pad,
```

Comment:
I think the file ops should be generated from the whatever handle holds the S3 credentials

Reply:
I'm not too sure I understand this one. I will add documentation about how the AWS credentials are setup, and then let's discuss some more. See my comments further down about what I was hoping to achieve with file_ops...

Reply:
I think the way to did this was right in the end!